lms/djangoapps/branding/tests/test_api.py

```diff
@@ -5,7 +5,7 @@
 from django.test import TestCase
 
 import mock
-from branding.api import get_logo_url, get_footer
+from branding.api import get_logo_url, get_footer, get_enterprise_customer_logo_url
```

Comment:
does it make sense to hide `get_enterprise__customer_logo_url` behind `get_logo_url`?